lib/control-app.js

```diff
@@ -25,10 +41,15 @@ module.exports = function(config) {
 
     var build;
     // default builder is browserify which we provide
-    config.builder = config.builder || '../lib/builder-browserify';
+    config.builder = config.builder || defaultBuilder;
```

Comment:
Yeah it might not be too useful outside of our needs, but I think it makes sense to keep the API the same across the internal and exposed use cases.